testutils/ocispec.go

```diff
@@ -278,3 +305,27 @@ func AreNamespacesDeactivated(nsList []specs.LinuxNamespace, namespaces []specs.
 
 	return true
 }
+
+// PathListMatchRefMount checks that the path list holds exactly the mount destinations of the provided mount list
+func PathListMatchRefMount(mountPathList []string, refMounts []specs.Mount) bool {
```

Comment:
Looks like the tests don't cover this -- would it make sense to add a unit test just for this method?

Reply:
Will do.

Reply:
(it does)

Reply:
can we add more cases?  It seems to only hit the `return true` with empty for loop checking